### ISSUE TITLE
[Bug]: config file Spamming logs

### ISSUE THREAD

**Issue by Saereth** [Fuzss/easymagic#69]
### Mod Loader (Required)

NeoForge

### Minecraft Version(s) (Required)

1.21

### Mod Version(s) (Required)

EasyMagic-v21.0.2-1.21-NeoForge

### Other Mods Involved (Required)

Yes

### Notes (Required)

Running easy magic on a neoforge server is causing the logs to get repeatedly and endlessly spammed with:

```
12:15:14  [Warn] [net.neoforged.fml.config.ConfigTracker/CONFIG]: Configuration file /home/minecraft/mc-instances/mc-65824b7278fb0/config/easymagic-server.toml is not correct. Correcting 
12:15:14  [Info] [Puzzles Lib/]: Dispatching reloading event for config at easymagic-server.toml
```

### latest.log (Required)

https://gist.github.com/Saereth/1b42a3a9fbac5eb8bb6b1032864b7c77

**Comment by Zelioth** [comment]
I can also confirm this issue on NeoFroge 1.21. Seems to be generating new configs on load 
![image](https://github.com/user-attachments/assets/0b908a79-b51b-41e7-a30d-f6333144e106)

**Comment by Fuzss** [comment]
Fixed in PuzzlesLib v21.0.19.
